rgw/v2/tests/s3_swift/multisite_configs/test_versioning_objects_suspend_archive_haproxy.yaml

```diff
@@ -0,0 +1,20 @@
+# upload type: non multipart
+# script: test_versioning_with_objects.py
+# polarian CEPH-83575578
+# bugzilla: 2137596
+config:
+     haproxy: true
+     test_versioning_archive: true
+     user_count: 1
+     bucket_count: 2
+     objects_count: 3
```

Comment:
@viduship , could you please increase object count

Reply:
@hmaheswa, hi, My intention to have a small dataset was that the suspend versioning code is already exercised in cephci with 400 object versions. this would test functionality for 6 objects over 2 buckets.